### PR TITLE
Fix Snyk issues

### PR DESCRIPTION
**Pull request by rpothier**
### Desired Outcome

There should be no active Snyk issues.

### Implemented Changes

Updated test yaml files as recommended by Snyk.
Updated go.mod

### Connected Issue/Story

Resolves #[relevant GitHub issue(s), e.g. 76]

CyberArk internal issue ID: CONJSE-1752
### Definition of Done
*At least 1 todo must be completed in the sections below for the PR to be
merged.*

#### Changelog

- [ ] The CHANGELOG has been updated, or
- [ ] This PR does not include user-facing changes and doesn't require a
  CHANGELOG update

#### Test coverage

- [ ] This PR includes new unit and integration tests to go with the code
  changes, or
- [ ] The changes in this PR do not require tests

#### Documentation

- [ ] Docs (e.g. `README`s) were updated in this PR
- [ ] A follow-up issue to update official docs has been filed here: [insert issue ID]
- [x] This PR does not require updating any documentation

#### Behavior

- [ ] This PR changes product behavior and has been reviewed by a PO, or
- [ ] These changes are part of a larger initiative that will be reviewed later, or
- [x] No behavior was changed with this PR

#### Security

- [ ] Security architect has reviewed the changes in this PR,
- [ ] These changes are part of a larger initiative with a separate security review, or
- [x] There are no security aspects to these changes

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -192,6 +192,10 @@ replace golang.org/x/net v0.1.1-0.20221104162952-702349b0e862 => golang.org/x/ne
 
 replace golang.org/x/text v0.0.0-20170915032832-14c0d48ead0c => golang.org/x/text v0.4.0
 
+replace golang.org/x/net v0.4.0 => golang.org/x/net v0.7.0
+
+replace golang.org/x/net v0.0.0-20220425223048-2871e0cb64e4 => golang.org/x/net v0.7.0
```
Comment (john-odonnell) [inline go.mod:197]: Should these changes be accompanied with an update to `go.sum` that removes the entries for these versions?

Comment (john-odonnell) [reply go.mod:197]: LGTM otherwise

Comment (rpothier) [reply go.mod:197]: yes! thanks for the reminder.

Comment (rpothier) [reply go.mod:197]: on second look, looks like v0.7.0 is already in go.sum so no update is generated.

**Review by john-odonnell — APPROVED** [review]

**Pull request merged**